Makefile.Common

```diff
@@ -32,3 +32,8 @@ lint:
 .PHONY: mod-tidy
 mod-tidy:
 	go mod tidy
+
+.PHONY: update-collector-ver
+update-collector-ver:
+	awk -v CORE="$(CORE_VER)" -v CONTRIB="$(CONTRIB_VER)" '/go.opentelemetry.io\/collector/ {sub("v[0-9]+\\.[0-9]+(\\.[0-9]+)?", CORE)} /github.com\/open-telemetry\/opentelemetry-collector-contrib/ {sub("v[0-9]+\\.[0-9]+(\\.[0-9]+)?", CONTRIB)} 1' \
```

Comment:
This works but has one issue. It updates `go.opentelemetry.io/collector/pdata v1.0.0-rcv0014 // indirect` to `	go.opentelemetry.io/collector/pdata v0.83.0-rcv0014 // indirect`. I couldn't get a regex to work for this case. If someone has ideas or is a better regex wizard please suggest changes.